Mothercell/Hive.IO/Hive.IO/DataHandling/Results.cs

```diff
@@ -72,7 +72,7 @@ public class Results
         /// </summary>
         public bool[,] SupplyTypes { get; private set; }
         /// <summary>
-        /// Capacity per technology. Unit is defined in 'SupplyCapUnits
+        /// Capacity per technology. UnitText is defined in 'SupplyCapUnits
```

Comment:
same as previous comment

Reply:
ugh! that was a refactoring gone wrong... I'll fix it.

Reply:
I reversed it